test/SIL.XForge.Scripture.Tests/Services/SFProjectServiceTests.cs

```diff
@@ -575,16 +644,10 @@ public async Task CheckLinkSharingAsync_ShareDisabledAndKeyValid_UserJoined()
         );
         project = env.GetProject(Project03);
         Assert.That(project.CheckingConfig.ShareEnabled, Is.False, "setup");
-        await env.Service.CheckLinkSharingAsync(User03, Project03, "key1234");
+        await env.Service.CheckLinkSharingAsync(User03, "key1234");
 
         project = env.GetProject(Project03);
-        projectSecret = env.ProjectSecrets.Get(Project03);
         Assert.That(project.UserRoles.ContainsKey(User03), Is.True, "User should have been added to project");
-        Assert.That(
```

Comment:
Maybe instead of just removing assertions that the key was removed, there should be assertions that the recepientId was set?